### DISCUSSION TITLE
Shelly Plus 1 ESP32 KNX stops working after some time on receiving telegrams

### DISCUSSION THREAD

**Discussion by ReelHiJacker** [arendst/Tasmota#19855] [Q&A]
I've got a problem with my two Shelly Plus 1's:

Both have the newest Tasmota (13.2) installed and i'm using KNX for switching On and Off. When the device is restarted everything works fine. But after some time the Shelly didn't receive any KNX Telegrams! The other way works all the time, which means that when i switch the shellys in the webinterface on or off, the knx bus gets the information from the shelly. But the shellys did not receive the commands anymore until i restart them. 

I tried differrent things:
1. Tried some Shelly 1 (ESP8266), they work all the time. Tried it in the same location, same network! Only the Shelly Plus 1 make problems.
2.  All, the KNX Router and the Shellys are on the same subnet. Also checked my Unifi Network Equipment, tried different settings, nothing helps.
3.  Tried different firmware versions, everytime the same problem.
4. Tried to log information on the shellys, found nothing
5. Tried something with Dynamic / Normal Sleep (https://tasmota.github.io/docs/Dynamic-Sleep/) also no change.
6. KNX and network works like a charm with my other different Tasmota Devices, no one except the shelly plus 1 had problems. So i think it's not a network - knx problem...

So for me it looks like there is something with the ESP32 Chip and the tasmota firmware which stops working after some time. But i can't find any solution. Did everyone have the same bevhavior or anyone with a similar setup but has no problem with knx?

**Comment by ReelHiJacker** [comment]
<img width="1018" alt="Bildschirmfoto 2023-10-28 um 08 23 51" src="https://github.com/arendst/Tasmota/assets/52639588/81e47502-7e2c-4583-ba57-eca275f979a5">
This is my knx setting, nothing special.

**Reply by Noschvie** [reply]
Willing to help with testing, will setup a plus 1 and send KNX messages from Node-Red to toggle the relay...

**Comment by Noschvie** [comment]
The test is up and running:
Tasmota 13.2.0, [tasmota32solo1.bin](https://ota.tasmota.com/tasmota32/release/tasmota32solo1.bin)

```
11:13:49.834 KNX: Received from 10/6/1 Command Write: 1 to Output 1
11:13:49.847 KNX: Output 1 = 1 sent to 10/6/11
11:13:49.853 MQT: stat/tasmota_AB07E0/RESULT = {"POWER":"ON"}
11:13:49.856 MQT: stat/tasmota_AB07E0/POWER = ON
11:14:51.158 KNX: Received from 10/6/1 Command Write: 1 to Output 1
11:14:51.173 KNX: Output 1 = 1 sent to 10/6/11
11:14:51.178 MQT: stat/tasmota_AB07E0/RESULT = {"POWER":"ON"}
11:14:51.182 MQT: stat/tasmota_AB07E0/POWER = ON
```

**Reply by ReelHiJacker** [reply]
That looks good, thank you for your help. I can not say how long it's working, i tried it some hours later, but i did not enclouse it at this time...

**Comment by Noschvie** [comment]
The test is running without problems and is going to continue...

**Reply by Noschvie** [reply]
The test is running without problems about 24 hours and is going to continue...

**Reply by ReelHiJacker** [reply]
Ok thanks, my system didn't run so long, only some hours until the receive problem occurs. So it seems not to be a general problem. Hm, i don't know what to do. The strange thing is that the normal shelly 1's are working on the same place with the same conditions. Overall i have 2 shelly plus 1 on two different points and both have the same problem.
